tests/test_cli.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: MIT
+
+import sys
+import unittest
+from argparse import Namespace
+
+import odxtools.cli.find as find
+import odxtools.cli.list as list_tool
+
+import_failed = False
+
+try:
+    import odxtools.cli.browse as browse
+except:
+    import_failed = True
+
+
+class UtilFunctions():
```

Comment:
the brackets look strange to me. (If you prefer them, you can leave them, though...)